agent/check_handler_env_test_windows.go

```diff
@@ -29,7 +30,7 @@ func TestEnvVars(t *testing.T) {
 	agent.sendq = ch
 
 	entity := agent.getAgentEntity()
-	agent.executeCheck(request, entity)
+	agent.executeCheck(context.TODO(), request, entity)
```

Comment:
Why `context.TODO()` here and `context.Background()` in the unix version?

Reply:
No particular reason